vignettes/generic.Rmd

```diff
@@ -215,13 +215,15 @@ dataset_vignette(
 ```
 
 Some further examples of "merged" functions are `derive_vars_merged_lookup()` to
-join a user-defined lookup table as commonly used in BDS ADaMs, or
-`derive_var_merged_cat()` to create a categorization variable from the information
-joined from the additional dataframe.
+join a user-defined lookup table as commonly used in BDS ADaMs,
+`derive_var_merged_cat()` to create a categorization variable from the
+information joined from the additional dataframe, or
+`derive_var_merged_summary()` to merge summarized values from the additional
```

Comment:
Could we make a new issue for someone to a more in-depth example of `derive_var_merged_summary()`.   I feel like this is a very elegant function that people will use a lot!!

Reply:
i'm on the fence with this one - not really sure! my thinking with this vignette was not to include an example for every one as i wanted people to still refer to the function reference pages themselves, but it was more just to get the general idea of why we have these different sets of functions and which you should look towards in difference scenarios. if you both feel this function will be commonly used though then find to add another example, e.g. maybe a total number of doses received variable in ADSL or something like that might work in keeping with the other similar examples used.

Reply:
I would not update the vignette now because I expect there will be major changes anyway when we implement https://github.com/pharmaverse/admiral/issues/1522.